cluegonauts/clueless/templates/clueless/gameb.html

```diff
@@ -0,0 +1,66 @@
+<!-- clueless/templates/clueless/index.html -->
+{% extends 'clueless/base.html' %}
+{% load static %}
+
+{% block content %}
+
+<!-- Main Content -->
+    <div class="container mx-auto">
+      <div class="grid grid-cols-3">
+          <div class="col-span-2">
+              <div class="label">
+                  <span class="text-xl">
+                      What would you like to do:
+                  </span>
+              </div>
+              <select id="player-select" class="select select-primary w-full max-w-xs" control-id="ControlID-1">
+                  <option value="Move to Study">Move to Study</div>
+                  <option value="Move to Ballroom">Move to Ballroom</div>
+                  <option value="Move to Billiards">Move to Billiards Room</div>
+                  <option value="Move to Dining Room">Move to Dining Room</div>
+                  <option value="Move to Hallway">Move to Hallway</div>
+                  <option value="Move to Kitchen">Move to Kitchen</div>
+                  <option value="Move to Lounge">Move to Lounge</div>
+                  <option value="Move to Conservatory">Move to Conservatory</div>
+                  <option value="Move to Library">Move to Library</div>
+                  <option value="action item">Make a suggestion</div>
+                  <option value="action item">Make a Accusation</div>
+              </select>
+              <div class="label">
+                  <button id="player-submit" class="btn btn-neutral"> Select Action </button>
+              </div>
+              <div id="game board" class="grid grid-rows-3 grid-flow-col gap-5 font-mono text-white text-sm text-center font-bold leading-6 rounded-lg">
```

Comment:
I think this looks great as is - we can add hallways along with graphics, in the minimal increment and let his be just the 3x3 grid for now I think!